src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { render, cleanup, wait, waitForDomChange } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../../services/pure-di';
+import ReportsPartialsCampaigns from './ReportsPartialsCampaigns';
+
+describe('ReportsPartialsCampaigns component', () => {
+  afterEach(cleanup);
+
+  it('should show the error message', async () => {
+    // Arrange
+    const dopplerApiClientDouble = {
+      getCampaignNameAndSubject: async () => {
+        return { success: false };
+      },
+      getCampaignSummaryResults: async () => {
+        return { success: false };
+      },
+    };
+    // Act
+    const { getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerApiClient: dopplerApiClientDouble,
+        }}
+      >
+        <IntlProvider>
+          <ReportsPartialsCampaigns />
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+    // Assert
+    await wait(() => expect(getByText('common.unexpected_error')).toBeInTheDocument());
+  });
+
+  it('should show feature not available message', async () => {
+    // Arrange
+    const dopplerApiClientDouble = {
+      getCampaignNameAndSubject: async () => {
+        return { success: true, value: { name: 'test', subject: 'test' } };
+      },
+      getCampaignSummaryResults: async () => {
```

Comment:
Are you sure that in case this is not implemented in prod this error is the correct one?

Reply:
yes! the error is for the `getCampaignSummaryResults` only because the database changes only affect this method